packages/roosterjs-editor-types/lib/interface/IEditor.ts

```diff
@@ -215,7 +215,8 @@ export default interface IEditor {
         clipboardData: ClipboardData,
         pasteAsText?: boolean,
         applyCurrentFormat?: boolean,
-        pasteAsImage?: boolean
+        pasteAsImage?: boolean,
+        nativePaste?: boolean
```

Comment:
Instead of adding new parameter here, how about put it into ClipboardData, so that we don't polute this function interface.